configs/test/project.yaml

```diff
@@ -118,3 +118,6 @@ env:
   # REVISION_VARS_URL:
 
   # Any of the above variables can be overridden in a job definition.
+
+bisect_service:
```

Comment:
Should this be commented ? Will this result in error if not used or unnecessary messages accumulating.

Reply:
Done. Commented to be safe.